osquery/tables/networking/windows/interfaces.cpp

```diff
@@ -0,0 +1,131 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <sstream>
+#include <string>
+
+#include <stdlib.h>
+
+#include <boost/algorithm/string/join.hpp>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/tables/system/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+void genAddresses(QueryData& results_data) {
+  std::stringstream ss;
+  ss << "SELECT * FROM win32_networkadapterconfiguration where IPEnabled=TRUE";
```

Comment:
The input to `WmiRequest` is a `std::string`, can you just:
```cpp
WmiRequest request("SELECT * FROM win32_networkadapterconfiguration where IPEnabled=TRUE");
```

Creating a `std::stringstream` is unneeded overhead.